docs/source/api/v1/statuses_id.rst

```diff
@@ -21,6 +21,9 @@
 
 ``GET``
 =======
+.. deprecated:: ATCv4
+	Use the ``GET`` method of :ref:`to-api-v1-statuses` with the query parameter ``id`` instead.
```

Comment:
Shouldn't this be `to-api-statuses`

Reply:
if the replacement route is in v1 I usually point that to them as they are are probably using v1 anyway

Reply:
Gothca